src/crawler/crawler.service.ts

```
@@ -1,150 +1,360 @@
+            location,
+            description,
+            applyUrl,
+            tags,
+            postedAt,
+            source: "WeWorkRemotely",
+            type: "Remote",
+          });
+        } catch (jobErr) {
+          this.logger.warn(`Failed to scrape a job on WWR: ${jobErr.message}`);
+          continue;
+        }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)